buildbot/configure.py

```diff
@@ -51,6 +51,7 @@ def do_configure(args):
 
     if sys.platform != "darwin":
         sycl_enabled_plugins.append("level_zero")
+        sycl_enabled_plugins.append("unified_runtime")
```

Comment:
```suggestion
```
I think this is not required at this point because unified_runtime is not a real plugin yet.